tests/functional/adapter/test_basic_iceberg.py

```diff
@@ -76,42 +74,42 @@ def configure_models_to_use_iceberg(models):
     return {key: configure_single_model_to_use_iceberg(val) for key, val in models.items()}
 
 
-class TestIcebergMergeNonUniqueLocation:
```

Comment:
could you totally remove this code? we don't really need to comment it out, if we need to refer back to it we can just ping point back to this PR.